docs/particles/ionization_states.rst

```diff
@@ -5,48 +5,49 @@ Ionization state data structures
 
 The ionization state (or charge state) of a plasma refers to the
 fraction of an element that is at each ionization level.  For example,
-the ionization state of a pure helium plasma could be 5% He⁰⁺, 94% He¹⁺,
-and 1% He²⁺.
+the ionization state of a pure helium plasma could be 5%
+He\ :sup:`0+`\ , 94% He\ :sup:`1+`\ , and 1% He\ :sup:`2+`\ .
 
 The ionization state of a single element
 ========================================
 
-We may use the `~plasmapy.particles.IonizationState` class
-to represent the ionization state of a single element, such as for this
-example.
+We may use the `~plasmapy.particles.IonizationState` class to represent
+the ionization state of a single element, such as for this example.
 
 >>> from plasmapy.particles import IonizationState
 >>> ionization_state = IonizationState("He", [0.05, 0.94, 0.01])
 
 The ionization state for helium may be accessed using the
-``ionic_fractions`` attribute.  These ionic fractions correspond to the
-``integer_charges`` attribute.
+`~plasmapy.particles.IonizationState.ionic_fractions` attribute.
+These ionic fractions correspond to the
+`~plasmapy.particles.IonizationState.integer_charges` attribute.
```

Comment:
```suggestion
`~plasmapy.particles.ionization_state.IonizationState.ionic_fractions` attribute.
These ionic fractions correspond to the
`~plasmapy.particles.ionization_state.IonizationState.integer_charges` attribute.
```

Reply:
We've slowly been incorporating the full paths to the definitions to ensure cross-referencing does not break.  In fact, this will be necessary when PR #1105 is merged.